tests/MakersOfDenmark.WebAPI.Tests/SearchForMakerSpaceHandlerTest.cs

```diff
@@ -49,7 +49,7 @@ public async Task SearchForMakerSpace_NotFoundTest()
             var result = await handler.Handle(new SearchForMakerSpace("NameDoesntExist"));
 
             //Assert
-            Assert.Null(result);
+            Assert.Empty(result);
```

Comment:
man kan også bruge fluent assertions her:
```
result.Should().HaveCount(0);  // tæller indholdet i en tom liste
```

Reply:
Det er også mere konsistent med at jeg bruger FluentAssertions i de andre tests +1

Reply:
@jesp209i Det ville nok være mere optimalt, da det er mere konsistent med at jeg bruger FluentAssertions i de andre tests, men i min handler har jeg dette:

`if (makerSpaces.Count == 0) { return null; }`//der returneres `null`, så `result.Should().HaveCount(0);` giver en fejl i test.

Måske du synes, at det er bedre, at jeg returnerer noget andet end `null` i min handler i så fald?

Reply:
Har du prøvet helt at fjerne din if-sætning?
Hvis den ikke finder noget i DB, så laver den en liste med antallet 0 makerspaces :)

Reply:
God idé, decideret null check er unødvendigt og jeg kan evt. tjekke for .Count(0) i endpoint. +1